src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
```

Comment:
I'm sorry I didn't get you there. Do you mean hookspecs for different scan stages should be defined in separate files?